client/src/contact.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+
+const Contact = () => (
+  <Container>
+    <h1>The Proto Company Project</h1>
+    <h4>
+      A client-facing retail web-portal had become significantly outdated and had been proven
+      to be a source of decline in sales numbers. The Proto Company Project comprises a complete
+      redesign of the retail portal designed to address concerns and modernize the site.
+
+      The initial release will focus on providing the minimum viable product for the retail
+      application. Features implemented were constrained to the client experience.
+    </h4>
+    <p>
+      The product detail page will show relevant information for a single product in the catalogue.
+      Our catalogue is organized by products. One single product can be associated with many sizes
+      and styles which each result in unique SKUs. The product detail page will present items at the
+      product level. Further breakdown by style or size will only be reflected within the product
+      detail page.
+    </p>
+    <p>
+      The same product detail page will be shown for every product in the catalogue. Upon navigating
+      to the detail page or selecting a new product to display, the contents of the page will update
+      to show information relevant to the selected product.
+    </p>
+    <p>
+      The item detail page will be comprised of distinct modules. Each will display information
+      related to the product being displayed. For the initial implementation, the following modules
+      have been prioritized:
+      <li>Product Detail - Critical Priority</li>
+      <li>Ratings & Reviews - High Priority</li>
+      <li>Questions & Answers - High Priority</li>
+      <li>Related Items & Outfit Creation - High Priority</li>
+    </p>
```

Comment:
YOU. ARE. AN. ANIMAL. AMAZING.

Reply:
🦖  Thank you! I felt this will be a great page to have when sending out our portfolios 👍